src/components/CodeView/utils.tsx

```diff
@@ -34,7 +34,7 @@ export const mapWithDepth = (depth: number, _mapChild = mapChild) => {
 };
 
 export const getLines = (content: string) => {
-  return content.replace('\r\n', '\n').split('\n');
+  return content ? content.replace('\r\n', '\n').split('\n') : [];
```

Comment:
Hmm. I don't think we need this patch anymore. When I load https://code.addons-dev.allizom.org/en-US/compare/616913/versions/1688316...1688317/?path=META-INF%2Fcose.sig (from master), everything looks OK. Am I missing anything?

Reply:
Aha, I see that we do need the patch (for the browse page)